oeps/oep-0002.rst

```diff
@@ -47,7 +47,7 @@ Specification
 Each repo in the Open edX system will include a file ``openedx.yaml``, with the
```

Comment:
Is there something that enforces these ``openedx.yaml`` files conform to the specifications below? If so, I will need to update it.